### ISSUE TITLE
Clarify manual for Bass module, OSC input

### ISSUE THREAD

**Issue by jimallman** [NikolaiVChr/Autinn#13]
I was getting poor results when passing in V/Oct here. After experimenting and reading more closely, I gather this input should be an audio signal (output of another oscillator "upstream"). It might be best to clarify this:

> Send the tone into OSC input.

And now I see that this is tagged as `filter` and not `oscillator`, so maybe that's just my ignorance showing.

**Comment by NikolaiVChr** [comment]
Yes, it works kinda like a TB-303, except you must supply the oscillator as external input. I did it that way to give people more options for getting some interesting sounds.

You can try see this video where I do that, first part of it is is one type of oscillator (Saw), second part is another (XCO).
https://www.youtube.com/watch?v=3gK3m_QrqfA

I will try to formulate the wiki better.
